arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts

```diff
@@ -0,0 +1,395 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qcom-apq8064-v2.0.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mfd/qcom-rpm.h>
+#include <dt-bindings/pinctrl/qcom,pmic-gpio.h>
+/ {
+	model = "LGE Nexus 4 (mako)";
+	compatible = "lge,nexus4-mako", "qcom,apq8064";
+
+	aliases {
+		serial0 = &gsbi7_serial;
+		serial1 = &gsbi6_serial;
+		serial2 = &gsbi4_serial;
+	};
+
+	chosen {
+		stdout-path = "serial2:115200n8";
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		ramoops@88d00000{
+			compatible = "ramoops";
+			reg = <0x88d00000 0x100000>;
+			record-size	 = <0x00020000>;
+			console-size	= <0x00020000>;
+			ftrace-size	 = <0x00020000>;
+		};
+	};
+
+	gpio-keys {
```

Comment:
does these keys work?

Reply:
I'm not sure, didn't test them yet. Removed for consistency.